pmd-apex/src/main/resources/rulesets/apex/ruleset.xml

```diff
@@ -12,6 +12,7 @@
    <rule ref="rulesets/apex/quickstart.xml/TooManyFields" deprecated="true" />
    <rule ref="rulesets/apex/quickstart.xml/AvoidDeeplyNestedIfStmts" deprecated="true" />
    <rule ref="rulesets/apex/quickstart.xml/CyclomaticComplexity" deprecated="true" />
+   <rule ref="rulesets/apex/quickstart.xml/AvoidDatabaseMethodCallsInLoops" deprecated="true" />
```

Comment:
Please also not modify this ruleset, it's deprecated in favor of quickstart.xml.... We probably should have added this info to the ruleset's description....